sdks/java/io/iceberg/src/main/java/org/apache/beam/sdk/io/iceberg/IcebergWriteSchemaTransformProvider.java

```diff
@@ -54,7 +54,7 @@ public class IcebergWriteSchemaTransformProvider
     extends TypedSchemaTransformProvider<Configuration> {
 
   static final String INPUT_TAG = "input";
-  static final String OUTPUT_TAG = "output";
+  static final String SNAPSHOTS_TAG = "snapshots";
```

Comment:
This is a breaking change that I'm comfortable with because I doubt anyone is fetching snapshots right now. I'd like to take the opportunity to give snapshots a more accurate tag and make room for a potential DLQ in the future